fix: bad internal id link in cascade_layers doc

wesleyboar opened the pull request:
### Description

Correct a bad internal link to a heading ID.

### Motivation

Link doesn't work. I think I know where it is meant to go.

### Additional details

N/A

### Related issues and pull requests

N/A

wesleyboar committed:
fix: bad internal id link in cascade_layers doc

wesleyboar commented on files/en-us/learn/css/building_blocks/cascade_layers/index.md:
```diff
@@ -288,7 +288,7 @@ In the first line, we import `components-lib.css` into the `components` layer. I
 The second line imports `narrowtheme.css` into the `narrow` layer, which is a sub-layer of `components`. The nested `components.narrow` gets created as the last layer within the `components` layer, unless `components-lib.css` already contains a `narrow` layer, in which case, the contents of `narrowtheme.css` would be appended to the `components.narrow` nested layer. Additional nested named layers ...
 
-Let's look at another example, where we import [`layers1.css`](#anonymous-and-named-layer-block-at-rule-assignment) into a named layer using the following statement:
+Let's look at another example, where we import [`layers1.css`]([#the_layer_block_at-rule_for_named_and_anonymous_layers) into a named layer using the following statement:
```
I had an extra punctuation. I should delete it.

```suggestion
Let's look at another example, where we import [`layers1.css`](#the_layer_block_at-rule_for_named_and_anonymous_layers) into a named layer using the following statement:
```

wesleyboar committed:
fix: remove extra and misplaced "[" in markdown link

wesleyboar committed:
Merge branch 'main' into wesleyboar-patch-2

estelle commented on files/en-us/learn/css/building_blocks/cascade_layers/index.md:
```diff
@@ -288,7 +288,7 @@ In the first line, we import `components-lib.css` into the `components` layer. I
 The second line imports `narrowtheme.css` into the `narrow` layer, which is a sub-layer of `components`. The nested `components.narrow` gets created as the last layer within the `components` layer, unless `components-lib.css` already contains a `narrow` layer, in which case, the contents of `narrowtheme.css` would be appended to the `components.narrow` nested layer. Additional nested named layers ...
 
-Let's look at another example, where we import [`layers1.css`](#anonymous-and-named-layer-block-at-rule-assignment) into a named layer using the following statement:
+Let's look at another example, where we import [`layers1.css`](#the_layer_block_at-rule_for_named_and_anonymous_layers) into a named layer using the following statement:
```
```suggestion
Let's look at another example, where we [import `layers1.css` into a named layer](#the_layer_block_at-rule_for_named_and_anonymous_layers) using the following statement:
```

estelle replied:
expanding the text of the link because the link seemed like it will go to a CSS file.

wesleyboar committed:
style: more accessible link text

This text better describes the target. (The target was never just `layers1.css`.)

Co-authored-by: Estelle Weyl <estelle@openwebdocs.org>

estelle reviewed (approved):
Thank you!!!
